client/state/posts/likes/reducer.js

```diff
@@ -120,10 +120,7 @@ export const itemReducer = withSchemaValidation(
 	}
 );
 
-const postIdReducer = keyedReducer( 'postId', itemReducer );
-postIdReducer.hasCustomPersistence = true;
-export const items = keyedReducer( 'siteId', postIdReducer );
-items.hasCustomPersistence = true;
+export const items = keyedReducer( 'siteId', keyedReducer( 'postId', itemReducer ) );
```

Comment:
I'm not sure why this was like that before - was it just to disable persistence?

Reply:
We needed to assign the intermediate reducers (like `postIdReducer`) to a variable in order to set the `hasCustomPersistence` flag on it. Now we can inline everything into one call.

A persisting reducer must have the `hasCustomPersistence` flag so that `combineReducers` doesn't ignore it. Only the `withSchemaValidation` wrapper used to add it. Other reducers needed to add it manually.

Now `keyedReducer` also adds it, correctly handling all kinds of inner reducers.

Reply:
Thought so, thanks for confirming 👍